back/spec/factories/portfolios.rb

```diff
@@ -0,0 +1,24 @@
+FactoryBot.define do
+  factory :portfolio do
+    sequence :name, 'portfolio_1'
+    sequence :url, 'url_1'
+    introduction { "This is a sample portfolio introduction." }
+    unhealthy_cnt { 0 }
+    latest_health_check_time { Time.now }
```

Comment:
Time.nowはUTCになってしまうのでcurrent使う癖つけといたほうがいいと思います！👍
```suggestion
    latest_health_check_time { Time.current }
```